.github/pull_request_template.md

```diff
@@ -2,14 +2,14 @@
 
 _Please be concise and link any related issue(s):_
 
-### 🚨Before submitting this pull request🚨:
+<!--### 🚨Before submitting this pull request🚨:-->
 
-_Please do **NOT** submit this PR if you have not done the following:_
+<!--_Please do **NOT** submit this PR if you have not done the following:_-->
 
-1. I have checked that no one else is working on similar changes.
-2. I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
-3. The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
-4. This branch is rebased/merged with the **latest master**.
-5. There are no merge conflicts.
-6. There are no console warnings and errors.
-7. On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
+- [] I have checked that no one else is working on similar changes.
+- [] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
+- [] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
+- [] This branch is rebased/merged with the **latest master**.
+- [] There are no merge conflicts.
+- [] There are no console warnings and errors.
+- [] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
```

Comment:
You need a space between the brackets so that way the checkbox is displayed in markdown. And do we want to have bullet points too? if not then probably remove the + sign

Reply:
updated

will appear like this - https://github.com/FightPandemics/FightPandemics-iOS/pull/179

Reply:
we need the bullet points to display properly

without the bullet points appears like this - https://jmp.sh/8AfXCWj